src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -51,7 +62,7 @@ const propTypes = {
     ...windowDimensionsPropTypes,
 };
 
-const defaultProps = fullPolicyDefaultProps;
+const defaultProps = {...fullPolicyDefaultProps, network: {isOffline: false}};
```

Comment:
Please expand this in the next lines to follow the codebase. 
```suggestion
const defaultProps = {
	...fullPolicyDefaultProps, 
	network: {isOffline: false},
};
```

Reply:
okay